build_deploy.sh

```diff
@@ -2,6 +2,20 @@
 
 set -exv
```

Comment:
Not related to the PR, but what's the reasoning for `-xv` here? It really makes the output and error messages you've added pretty difficult to see.

From the man page:

```
-v      Print shell input lines as they are read.
-x      After expanding each simple command, for command, case command, select command, or arithmetic for command, display the expanded value of PS4, followed by the command and its  expanded
        arguments or associated word list.
```

Without them, the script is nice and easy to figure out:

```
$ ./build_deploy.sh
RH_REGISTRY_USER and RH_REGISTRY_TOKEN must be set
```

Reply:
It is good for logging (and audit), as you can see what is actually happening. Not sure if `-x` sends it to `stdout`, but if it does we can print error messages to `stderr` to be able to look for them.

Reply:
I think it's because this is meant to be run on Jenkins and they want all the possible output. grabbed it from the existing templates for pr_check.sh  script. I'm also not a fan.

From Bash manpage
> -e      Exit immediately if a pipeline (which may consist of a single simple command), a list, or a compound command (see SHELL GRAMMAR above), exits with a non-zero status. [ ... ]
> -v      Print shell input lines as they are read.
> -x      After expanding each simple command, for command, case command, select command, or arithmetic >for command, display the expanded value of PS4, followed by the command and its expanded arguments or >associated word list.

Reply:
@akofink , @vkrizan  I've refactored this set flags to change it's behavior if it's a local run or not , hope it's okay